bin/ingest-convert.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+java -cp "$(cd `dirname $0`/..; pwd)"'/tools/ingest-converter/build/libs/ingest-converter.jar:*' \
```

Comment:
We don't bundle `tools` in our release package. Can you add this dir to https://github.com/elastic/logstash/blob/master/rakelib/artifacts.rake#L8? I would add `tools/ingest-converter/build/libs/*.jar`. We'll then have access to this location.

Also, I tried running

```sh
gradle clean assemble
bin/ingest-convert.sh 
``` 
and it threw:

```
Exception in thread "main" java.lang.NoClassDefFoundError: joptsimple/OptionSpec
	at org.logstash.ingest.Pipeline.main(Pipeline.java:15)
Caused by: java.lang.ClassNotFoundException: joptsimple.OptionSpec
	at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:335)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 1 more
```